docs/src/main/docs/tracing/01_tracing.adoc

```diff
@@ -57,3 +57,72 @@ ServerConfiguration.builder()
 ----
 <1> The name of the application (service) to associate with the tracing events
 <2> The endpoint for tracing events, specific to the tracer used, usually loaded from Config
+
+=== Configuring Tracing with Helidon MP
+There is an additional dependency that adds Tracing support to MP server and Jersey client:
+
+[source,xml,subs="verbatim,attributes"]
```

Comment:
I know this is used everywhere, `subs="verbatim,attributes` is not needed here (and in most cases).
Will need to do a pass to clean this up everywhere in the docs, but let's not add more ;)

Reply:
removed